hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/HoodieIncrSource.java

```diff
@@ -380,8 +380,7 @@ private Pair<Option<Dataset<Row>>, Checkpoint> fetchNextBatchBasedOnRequestedTim
           .option(START_COMMIT().key(), queryInfo.getStartInstant())
           .option(END_COMMIT().key(), queryInfo.getEndInstant())
           .option(INCREMENTAL_FALLBACK_TO_FULL_TABLE_SCAN().key(),
-              props.getString(INCREMENTAL_FALLBACK_TO_FULL_TABLE_SCAN().key(),
-                  INCREMENTAL_FALLBACK_TO_FULL_TABLE_SCAN().defaultValue()))
+              props.getString(INCREMENTAL_FALLBACK_TO_FULL_TABLE_SCAN().key(), "false"))
```

Comment:
why flipping the def value even for tbl version 8?

Reply:
The change is made for function `fetchNextBatchBasedOnRequestedTime` which is used for table version 6.